ppr-ui/src/views/amendment/AmendRegistration.vue

```diff
@@ -651,6 +653,42 @@ export default class AmendRegistration extends Vue {
     this.debtorValid = val
   }
 
+  private validateSecuredParties (): void {
+    const sp = this.getAddSecuredPartiesAndDebtors.securedParties
+    let securedPartyCount = 0
+    if (this.registrationType === APIRegistrationTypes.SECURITY_AGREEMENT) {
+      for (let i = 0; i < sp.length; i++) {
```

Comment:
Could probably use a find() here, to simplify.

Reply:
Good Suggestion.  Find is not appropriate, but Filter works...